jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
     }
 
     options {
-        timeout(time: 300, unit: 'MINUTES')
```

Comment:
Looking back at this, I'm not sure I understand. That timeout is of 5 hours ! The CI job fails way before that...

Reply:
Does it? When I was looking at the jobs I thought it was failing due to the global Jenkins timeout, but maybe I mislooked.